src/ImGui.NET/ImGui.cs

```diff
@@ -896,7 +940,7 @@ public static void SameLine()
             ImGuiNative.igSameLine(0, 0);
         }
 
-        public static void SameLine(float localPositionX, float spacingW)
+        public static void SameLine(float localPositionX = 0, float spacingW = -1.0f)
```

Comment:
I think we should either

* Remove the parameterless overload above this
or
* Make the parameterless overload use the same "default values" as here.

Thoughts?